content/docs/command-reference/remote/modify.md

```diff
@@ -419,7 +454,7 @@ $ export AZURE_STORAGE_CONNECTION_STRING='mysecret'
 For account name and key/token auth:
 
 ```dvc
-$ export AZURE_STORAGE_ACCOUNT='myuser'
+$ export AZURE_STORAGE_ACCOUNT='mystorageaccount'
```

Comment:
Same Q. And no `_` (undersscores) here?

Reply:
See my first reply (where I will remove the underscores)

Reply:
OK got it, I'll reupdate these 👍